addons/portal/wizard/portal_wizard_views.xml

```diff
@@ -1,12 +1,19 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
         <!-- wizard action on res.partner -->
+        <record id="partner_wizard_action_create_and_open" model="ir.actions.server">
```

Comment:
@tde-banana-odoo This is needed, because we need to first create the record `portal.wizard` (and his `portal.wizard.user`) and then open it, otherwise the button will be disabled (because we can not execute a button action on a record which does not exist yet)